.eslintignore

```diff
@@ -0,0 +1 @@
+/assets/js/bundle.js
```

Comment:
Can you add the config files here as well, ESLint is operating under ES2016 guidelines